### DISCUSSION TITLE
what the data ratio was in training canary-1b-flash

### DISCUSSION THREAD

**Discussion by sh1man999** [NVIDIA-NeMo/NeMo#12996] [Q&A] [answered]
I'm interested in how the dataset was split during the training of the canary-1b-flash model. What percentage of data was allocated to the training set, what percentage to the validation set, and what percentage to the test set? Was it a standard ratio like 80/10/10 or was something different used?

**Comment by ankitapasad** [comment, marked as answer]
Hi @sh1man999 

Thank you for your interest!

Please refer to the [Canary paper](https://www.isca-archive.org/interspeech_2024/puvvada24_interspeech.pdf) for a detailed discussion of training and test data. For monitoring performance during training we use LibriSpeech and MCV for multilingual ASR and MUSTC and Europarl for AST.
